tools/wasm-sourcemap.py

```diff
@@ -198,12 +211,10 @@ def read_dwarf_entries(wasm, options):
 
   entries = []
   debug_line_chunks = re.split(r"debug_line\[(0x[0-9a-f]*)\]", output.decode('utf-8'))
-  maybe_debug_info_content = debug_line_chunks[0]
+  map_stmt_list_to_comp_dir = parse_debug_info_contents(debug_line_chunks[0])
   for i in range(1, len(debug_line_chunks), 2):
     stmt_list = debug_line_chunks[i]
```

Comment:
Unrelated to this PR, but it looks like these two lines can be written as:

```
  for stmt_list in debug_line_chunks[1::2]:
```

Reply:
Ok, I included this change in the PR.